.github/workflows/build.yml

```diff
@@ -37,31 +37,45 @@ jobs:
               dnf -y install $dnf_opts \
                 git ${{ matrix.compiler }} meson \
                 pkgconf-pkg-config openssl-devel openssl \
-                diffutils expect valgrind
+                diffutils expect valgrind opensc gnutls-utils
               if [ "${{ matrix.token }}" = "softokn" ]; then
-                dnf -y install nss-softokn nss-tools nss-softokn-devel
+                dnf -y install nss-softokn nss-tools nss-softokn-devel \
+                  nss-devel
               elif [ "${{ matrix.token }}" = "softhsm" ]; then
-                dnf -y install softhsm opensc p11-kit-devel p11-kit-server \
-                  gnutls-utils
+                dnf -y install softhsm p11-kit-devel p11-kit-server
```

Comment:
once we touch it, we could probably remove the `p11-kit-server` from this line as we no longer use the p11-kit remoting. We should probably remove it from the `packaging/pkcs11-provider.spec` too.

Reply:
ok

Reply:
should we also remove the whole softhsm-proxy.sh thing ?

Reply:
Yes. I think it only confuses things. But note that the p11-kit itself should stay around for the default proxy module.

Reply:
Done, PTAL